src/components/literacy-material/literacy-material.resolver.ts

```diff
@@ -2,79 +2,73 @@ import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
 import { AnonSession, ID, IdArg, LoggedInSession, Session } from '../../common';
 import { Loader, LoaderOf } from '../../core';
 import {
-  CreateLiteracyMaterialInput,
-  CreateLiteracyMaterialOutput,
-  LiteracyMaterial,
-  LiteracyMaterialListInput,
-  LiteracyMaterialListOutput,
-  UpdateLiteracyMaterialInput,
-  UpdateLiteracyMaterialOutput,
-} from './dto';
-import { LiteracyMaterialLoader } from './literacy-material.loader';
-import { LiteracyMaterialService } from './literacy-material.service';
+  CreateEthnoArtInput,
+  CreateEthnoArtOutput,
+  EthnoArt,
+  EthnoArtListInput,
+  EthnoArtListOutput,
+  EthnoArtLoader,
+  EthnoArtService,
+  UpdateEthnoArtInput,
+  UpdateEthnoArtOutput,
+} from '../ethno-art';
+import { LiteracyMaterial } from './dto';
 
+/** @deprecated Use EthnoArtResolver instead */
```

Comment:
The Query/Mutation/ResolveField decorators have a `deprecatedReason` property. Can you use this instead? This will communicate in the external schema that these are deprecated.